src/components/hoc/withMeeting.jsx

```diff
@@ -32,7 +33,9 @@ export default function withMeeting(WrappedComponent) {
     const {meetingDestination} = props;
     const meeting = useMeetingDestination(meetingDestination);
 
-    return <WrappedComponent meeting={meeting} {...props} />;
+    return <WebexMeetingProvider>
```

Comment:
`withMeeting` now ensures that the `MeetingContext` has been created during initialization. In the future, if we'd like to provide context to any of the components rendered, they can utilize the `WebexMeetingProvider` and `MeetingContext` without having to drill down through the components.